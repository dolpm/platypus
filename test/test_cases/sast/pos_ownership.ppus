pipe main |> [] |> [] |> unit {
  int x <| 4;
  while (x <= 4) {
    int y <| x;
  }
}