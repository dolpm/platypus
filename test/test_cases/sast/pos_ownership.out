pipe main [] |> unit {
{(id: main_with_args)
{(id: main_wrapper)
  int x = (int : 4);
{(id: 1)
  int y = (int : x);
}
  |> (unit : ());
}
}
}
